src/main/scala/com/bwsw/cloudstack/vault/server/util/Constants.scala

```diff
@@ -25,11 +25,16 @@ object ConfigLiterals {
   private final val zooKeeperDomain   = s"$applicationDomain.zookeeper"
   private final val cloudStackDomain  = s"$applicationDomain.cloudStack"
 
-  val tokenPeriod            = s"$applicationDomain.tokenPeriod"
-  val accountsVaultBasicPath = s"$applicationDomain.accountsVaultBasicPath"
-  val vmsVaultBasicPath      = s"$applicationDomain.vmsVaultBasicPath"
   val tagNamePrefix          = s"$applicationDomain.tagNamePrefix"
 
+  val vaultUrl               = s"$vaultDomain.url"
+  val vaultRootToken         = s"$vaultDomain.rootToken"
+  val vaultRetryDelay        = s"$vaultDomain.retryDelay"
+  val tokenPeriod            = s"$vaultDomain.tokenPeriod"
```

Comment:
make the variable names in one uniform style

Reply:
ok